src/domains/apiconfig-app/05_subkey.tf

```diff
@@ -0,0 +1,8 @@
+resource "azurerm_api_management_subscription" "apiconfig_cache_for_rtp_subkey" {
+  api_management_name = data.azurerm_api_management.apim.name
+  resource_group_name = data.azurerm_api_management.apim.resource_group_name
+  product_id          = module.apim_apiconfig_cache_external_product.id
```

Comment:
`id` in `apim_apiconfig_cache external` product does not exist. It should probably be `product_id`